doc/lua_api.txt

```diff
@@ -2385,6 +2386,10 @@ and `minetest.auth_reload` call the authetification handler.
     * Convert a vector to a wallmounted value, used for `paramtype2="wallmounted"`
 * `minetest.wallmounted_to_dir(wallmounted)`
     * Convert a wallmounted value back into a vector aimed directly out the "back" of a node
+* `minetest.dir_to_yaw(dir)`
+    * Convert a vector into a yaw (angle)
+* `minetest.yaw_to_dir(yaw)`
+    * Convert yaw (angle) to a vector
```

Comment:
@sofar these functions aren't entity functions thus I don't understand your argument. These are merely geometric functions. In fact you could replace both pitch and yaw by something like
angle_from_2d_direction({a1=?,b1=?}, {a2=?,b2=?})

but if you did you'd have to use correct input values for a and b so the yaw(pitch) thingy is more comfortable